spec/models/unpublishing_spec.rb

```diff
@@ -50,11 +50,36 @@
       expect(subject).to be_valid
     end
 
-    it "requires an alternative_path for a 'redirect'" do
+    it "requires a redirects blob for a 'redirect'" do
       subject.type = "redirect"
       subject.alternative_path = nil
+      subject.redirects = nil
```

Comment:
Looks like this came before you so don't worry if this seems awful. But it'd be good to alias subject to a more meaningful name (unpublishing perhaps?) and not use it explicitly